policy/modules/kernel/terminal.if

```diff
@@ -1299,6 +1299,42 @@ interface(`term_dontaudit_ioctl_unallocated_ttys',`
 	dontaudit $1 tty_device_t:chr_file ioctl;
 ')
 
+########################################
+## <summary>
+##	Watch unallocated tty device nodes.
+## </summary>
+## <param name="domain">
+##	<summary>
+##	Domain allowed access.
+##	</summary>
+## </param>
+#
+interface(`term_watch_unallocated_ttys',`
+	gen_require(`
+		type tty_device_t;
+	')
+
+	allow $1 tty_device_t:chr_file watch_file_perms;
```

Comment:
(cosmetic) There is already `watch_chr_file_perms` so it would be better to match the class. And `watch_reads_chr_file_perms` can be added and used below, too,

Reply:
Updated.